src/acktest/k8s/resource.py

```diff
@@ -109,10 +110,14 @@ def load_and_create_resource(resource_directory: Path,
     resource = create_resource(reference, spec)
     return reference, spec, resource
 
+
 def _get_k8s_api_client() -> ApiClient:
     # Create new client everytime to avoid token refresh issues
     # https://github.com/kubernetes-client/python/issues/741
     # https://github.com/kubernetes-client/python-base/issues/125
+    if os.environ.get('LOAD_IN_CLUSTER_CONFIG') is not None and os.environ.get('LOAD_IN_CLUSTER_CONFIG').lower() == 'true':
```

Comment:
This conditional can be greatly simplified to 
```suggestion
if bool(distutils.util.strtobool(os.environ.get('LOAD_IN_CLUSTER_CONFIG', "false")): 
```

Reply:
Thanks! I will test it out and then update the CR soon.

Reply:
> This conditional can be greatly simplified to

Done.